bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcComponentProvider.kt

```diff
@@ -65,18 +67,29 @@ class BcmcComponentProvider(
             analyticsService = analyticsService,
             analyticsMapper = AnalyticsMapper(),
         )
+
         val bcmcFactory = viewModelFactory(savedStateRegistryOwner, defaultArgs) { savedStateHandle ->
+            val bcmcDelegate = DefaultBcmcDelegate(
+                observerRepository = PaymentObserverRepository(),
+                paymentMethod = paymentMethod,
+                publicKeyRepository = publicKeyRepository,
+                componentParams = componentParams,
+                cardValidationMapper = cardValidationMapper,
+                cardEncrypter = cardEncrypter,
+                analyticsRepository = analyticsRepository,
+                submitHandler = SubmitHandler()
+            )
+
+            val genericActionDelegate = GenericActionComponent.PROVIDER.getDelegate(
```

Comment:
I think we should use `GenericActionComponentProvider(overrideComponentParams)` here.

My suspicion is that if we launch a component from drop-in, and set a custom config for it, the drop-in config will override the custom config for the component delegate but not for the action delegate. Can you make a quick test to see if this true?

Reply:
You are right, let me fix it